validator/lib/validateB3dm.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+var Cesium = require('cesium');
+var DeveloperError = Cesium.DeveloperError;
+module.exports = validateB3dm;
+
+/**
+ * Test if the provided data is gzipped.
+ *
+ * @param {Buffer} content A buffer containing the contents of a b3dm tileset.
+ * @returns {Boolean} True if the content is valid according to the specs:
+ * https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/TileFormats/Batched3DModel, False if not.
+ */
+function validateB3dm(content) {
+    if(content === undefined || content === null) {
+        throw new DeveloperError('b3dm content is undefined or null');
```

Comment:
Change the description so its a bit more directive, like `content must be defined`.